docs/guides/local-apt-repository.rst

```diff
@@ -113,7 +114,7 @@ you have your containers inside of a ``[containers]`` group.
 
 ``debops -l containers``
 
-Use your shiny new package
+Using your shiny new package
 --------------------------
```

Comment:
The underline needs to match the header, otherwise it won't be parsed correctly by Sphinx.